MapboxNavigation/CGPoint.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension CGPoint {
+    func CGPointDistanceSquared(from: CGPoint, to: CGPoint) -> CGFloat {
```

Comment:
This is an instance method in an extension, whereas the method is named as if it’s a standalone function. It’s weird that you can call `point1.CGPointDistanceSquared(from: point2, to: point3)`.

Replace `from` with `self` and remove “CGPoint” from the method name. Also reconsider whether it’s even necessary to break this method out separately.